benchmarks/pubmed_sentence_classifier/convert_to_classifier_input.py

```diff
@@ -33,14 +35,15 @@ def parse_raw_file(raw_file_path:Path)->Iterable[Record]:
   for doc in docs:
     for idx, line in enumerate(doc):
       try:
-        label, text = line.split(" ", 1)
+        label, text = line.split("\t", 1)
```

Comment:
This was a bug, turns out the raw data uses a tab to separate out the label from the text. Assert guarantees that things went okay.